src/main/kotlin/no/nav/familie/ef/sak/behandling/migrering/AutomatiskMigreringService.kt

```diff
@@ -0,0 +1,39 @@
+package no.nav.familie.ef.sak.behandling.migrering
+
+import no.nav.familie.ef.sak.infotrygd.InfotrygdReplikaClient
+import no.nav.familie.ef.sak.opplysninger.personopplysninger.secureLogger
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+
+@Service
+class AutomatiskMigreringService(private val migreringsstatusRepository: MigreringsstatusRepository,
```

Comment:
Kanskje vurdere å ha en metric på antall automatiske migreringer, men det er ikke viktig i denne PR-en, men fint for senere...

Reply:
Vet ikke om det gir veldig mye verdi å se diff mellom manuelle/automatiske. Når vi har migrert alle så trenger vi nog ikke de verdiene? 
Går eventuellt å legge inn det i den metrikken som finnes med en tag, eks å gjøre utplukk på de som har opprettet av VL på behandling og årsak = migrert eller noe sånt